odoo-openmrs/src/main/java/com/ozonehis/eip/odoo/openmrs/component/OdooEndpoint.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright © 2024, Ozone HIS <info@ozone-his.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.ozonehis.eip.odoo.openmrs.component;
+
+import com.ozonehis.eip.odoo.openmrs.client.OdooClient;
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.camel.*;
+import org.apache.camel.spi.Metadata;
+import org.apache.camel.spi.UriEndpoint;
+import org.apache.camel.spi.UriPath;
+import org.apache.camel.support.DefaultEndpoint;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Odoo component to integrate with Odoo XML RPC.
+ */
+@UriEndpoint(firstVersion = "1.0.0", scheme = "odoo", title = "Odoo", syntax = "odoo:method/model", producerOnly = true)
```

Comment:
Why not set the method as a header or param? I think having something like odoo:get/re.user is less intuitive.

Reply:
I think this is how Camel components are configured in their library
For Eg: FHIR Component
https://github.com/apache/camel/blob/main/components/camel-fhir/camel-fhir-component/src/main/java/org/apache/camel/component/fhir/FhirEndpoint.java#L55